File: mobile/src/types/chapter.ts
export interface EventChapter {
  id: string;
  chapterIndex: number;
  chapterTitle?: string | null;
  chapterStory?: string | null;
  chapterIntro?: string | null;
  chapterSummary?: string | null;
  slideshowCaption?: string | null;
  photoStartIndex: number;
  photoEndIndex: number;
  createdAt: string;
}

File: backend/app/schemas/chapter.py
from __future__ import annotations

from datetime import datetime
from typing import Optional

from pydantic import BaseModel


class EventChapterResponse(BaseModel):
    id: str
    chapterIndex: int
    chapterTitle: Optional[str] = None
    chapterStory: Optional[str] = None
    chapterIntro: Optional[str] = None
    chapterSummary: Optional[str] = None
    slideshowCaption: Optional[str] = None
    photoStartIndex: int
    photoEndIndex: int
    createdAt: datetime
